doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -33,7 +33,7 @@ The following sections provide detailed lists of changes by component.
 IDE, OS, and tool support
 =========================
 
-|no_changes_yet_note|
+* Added :ref:`nrf_toolchain_docker_image`, a containerized version of the |NCS| toolchain that can be used for development and testing.
```

Comment:
Will move this entry to the release notes PR https://github.com/nrfconnect/sdk-nrf/pull/18192, after the review is done here.